index.js

```diff
@@ -540,8 +540,8 @@
   //. The Maybe type represents optional values: a value of type `Maybe a` is
   //. either a Just whose value is of type `a` or a Nothing (with no value).
   //.
-  //. The Maybe type satisfies the [Monoid][], [Monad][], [Foldable][], and
-  //. [Extend][] specifications.
+  //. The Maybe type satisfies the [Monoid][], [Monad][], [Foldable][],
+  //. [Traversable][], and [Extend][] specifications.
```

Comment:
This came back to life during the rebase:

> Traversable implies Foldable, so we could remove Foldable from the list in the interest of brevity.

I believe this comment still applies:

> Can we remove the link definition for `[Foldable]`? I believe we're removing the only occurrence.